.pre-commit-config.yaml

```diff
@@ -16,26 +16,26 @@ repos:
       - id: check-added-large-files
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3 # v2.7.1
+    rev: ffb6a759a979008c0e6dff86e39f4745a2d9eac4 # v3.1.0
```

Comment:
This part confuses me. Your seems to explicitly skip over `mirrors-prettier` when I run it and with the line 
```python
if "mirrors-prettier" not in repo["repo"]:
```
but there is still an update here? Which is it?

Reply:
It will autoupdate to the latest tag (which is an alpha release):
https://github.com/pre-commit/mirrors-prettier/releases/tag/v4.0.0-alpha.8

Reply:
This is the last stable release:
https://github.com/pre-commit/mirrors-prettier/releases/tag/v3.1.0

Reply:
Got it.